src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,68 @@
+import React, { useEffect, useState } from 'react';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [load, setLoad] = useState(false);
+
+  const loadPeople = async () => {
+    setLoad(true);
+    const peoplesFromServer = await getPeople();
+    const peoples = peoplesFromServer.map(peopleFromServer => ({
+      ...peopleFromServer,
+      mother: peoplesFromServer.find(
+        person => person.name === peopleFromServer.motherName,
+      ),
+      father: peoplesFromServer.find(
+        person => person.name === peopleFromServer.fatherName,
+      ),
+    }));
+
+    try {
+      setPeople(peoples);
+    } catch {
+      setError(true);
+    } finally {
+      setLoad(false);
+    }
```

Comment:
Pointless use of the try/catch structure. Since you are unlikely to get an error setting the state